Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -136,6 +133,45 @@ def get_events_with_offset(
             demisto.error(f"couldn't decode offset with {offset_context=}, reason {e}")
         return events, offset
 
+    async def get_events_with_offset_concurrently(
+        self,
+        config_ids: str,
+        offset: str | None = '',
+        limit: int = 200000,
+        from_epoch: str = '',
+        counter: int = 0
+    ) -> tuple[list[str], str | None]:
+        """Send request to get events from Akamai.
+
+        Args:
+            config_ids (str): security configuration ids to fetch, e.g. `51000;56080`.
+            offset (str | None): The offset (hash) to use for offset based mechanism.
+            limit (int, optional): The number of events to limit for every request.
+            from_epoch (str): From when to fetch if first time.
+            counter (int, optional): The execution number.
+
+        Returns:
+            tuple[list[str], str | None]: The events and offset obtained from last request.
+        """
+        params = prepare_params(offset=offset, limit=limit, from_epoch=from_epoch,
```

Comment:
doesnt prepare params always need to happen inside `execute_get_events_request` ? how does this work in the non-async flow?
In general makes sense all the actual logic with the request should happen inside that function and get_events_with_offset_concurrently should really only wrap it up to make it fitting for asyncio